site/content/en/docs/integration/toloka.md

```diff
@@ -0,0 +1,213 @@
+     within which each job should be completed. You can also click on the
+     field to select from one of the predefined periods.
+   - **Time per task suite, sec**: Enter the time allowed for completing a task suite,
+     in seconds. Uncompleted tasks are redistributed to other Tolokers.
+   - **Days for manual review by requester**: Specify the Review period in days —
+     the number of days for the review (from 1 to 21 days from the task completion date).
+     The Toloker will see the deadline in the task information on the main Toloka page.
+   - **Audience**: Add rules to make jobs available only to Tolokers who meet certain criteria.
+     For example, you might require Tolokers to be proficient in English and have higher education.
+     These rules operate based on filter principles. For more information,
+     see [Filters](http://localhost:1313/docs/manual/advanced/filter/) documentation
+     or [CVAT Filters](https://www.youtube.com/watch?v=lj6KLIFn24A) video tutorial.
+   - ![Toloka Rules](/images/toloka10.jpg)
+
+3. Click **Submit**. You will see the **Toloka task was created** pop-up and the **Update pool** form.
+
+   ![Update Project](/images/toloka11.jpg)
+
+   - ![Open pool in Toloka](/images/toloka04.png) opens pool in Toloka.
+   - ![Open task in CVAT](/images/toloka05.png) opens task in CVAT.
+
+4. Open Toloka task, go to **Actions** > **Start Toloka pool**.
+   <br>Project, that you created will show up in Toloka.</br>
+
+   ![Toloka Project](/images/toloka12.jpg)
+
+## Changing Toloka pool
+
+To change started Toloka pool, you need to stop it first.
+
+1. Open Toloka task, **Actions** > **Stop Toloka pool**.
+2. Implement changes.
+3. Open Toloka task, go to **Actions** > **Start Toloka pool**.
+
+## Reviewing annotated jobs
+
+In case the jobs you've created are not in the **Accept solutions automatically**
+mode, you will need to manually review and accept them
+within time limits that were defined in the task setting.
+
+To approve or reject the job, use the **Accept** and **Reject** buttons.
+
+![Toloka Project](/images/toloka13.jpg)
+
+### Accepting job
+
+To accept the annotated job, do the following:
+
+1. Go to the Toloka task and open the job.
+2. Review the result of annotation and in case all is fine, on the top menu,
+   click **Accept**.
+3. Add comment.
+
+   ![Toloka Project](/images/toloka14.jpg)
+
+4. Click **OK**.
+
+### Rejecting job
+
+To accept the annotated job, do the following:
+
+1. Go to the Toloka task and open the job.
+   <br>On the top menu, you will see **Accept** and **Reject** buttons.
+   ![Toloka Project](/images/toloka13.jpg)
+
+2. Review the result of the annotation and in case something is wrong,
+   on the top menu, click **Reject**.
+3. Add comment.
+
+   ![Toloka Project](/images/toloka15.jpg)
+
+4. Click **OK**.
+
+After you reject the job, the menu bar will change and only the **Accept**
+button will be active, so you can accept the job when changes are made.
```

Comment:
Not exactly in this case. 
Tolokers will not have access to this job anyway, so they can't update any annotations. 
Just in case of appelation or changed desition, on any other case which we should not mention I believe. 
Maybe more general words like ``Rejected job can be accepted later`` would work